fjord/suggest/providers/trigger/templates/trigger/triggerrule_list.html

```diff
@@ -0,0 +1,46 @@
+{% extends "analytics/analyzer/dashboard.html" %}
+{# Note: This is not l10n-ized since it's only available to analyzers for now. #}
+
+{% block content_middle %}
+  <div class="block feedback">
+    <h2>Trigger Rules</h2>
+    <p>
+      <a href="{{ url('triggerrules-create') }}">Create new rule</a>
+    </p>
+    <p>Total trigger rules: {{ object_list.count() }}</p>
+    <table class="summarytable">
+      <tr>
+        <th>id</th>
+        <th>enabled?</th>
+        <th>title</th>
+        <th>versions</th>
+        <th>locales</th>
+        <th>products</th>
+        <th>url exists</th>
+        <th>keywords</th>
+        <th>actions</th>
+      </tr>
+      {% for obj in object_list %}
+        <tr>
+          <td>{{ obj.id }}</td>
+          <td>{{ obj.is_enabled }}</td>
+          <td>{{ obj.title }}</td>
+          <td>{{ obj.locales or '--' }}</td>
+          <td>
+            {% for prod in obj.products.all() %}
+              {{ prod.db_name }}{% if not loop.last %}, {% endif %}
+            {% else %}
+              --
+            {% endfor %}
+          </td>
+          <td>{{ obj.versions or '--' }}</td>
```

Comment:
In the body, the order is locales, products, versions. In the headers it is versions, locales, products. I think versions got misplaced in one of those.